src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -80,7 +80,6 @@ public async Task ExecuteResultAsync(ActionContext context)
             {
                 foreach (var cookie in Cookies)
                 {
-                    // Item3 (CookieOptions) should not be null, but this will behave correctly if it is
```

Comment:
nit: i think this comment still applies

Reply:
Noticed this comment after merging. I removed as it is a handled scenario and I din't feel it was providing much value to others working on the codebase. Is there something you feel others may be tempted to do if this comment is not present?

Side note, it's unfortunate, but there's a slight difference in behavior between the two Append overloads.

Reply:
That's a good point :) I mainly added it just to say this is not an expected code path in case someone is trying to reason through what the inputs look like, but of course that could change. So I think it's fair to remove it!